src/devices/Common/Iot/Device/Common/SimpleConsoleLogger.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Iot.Device.Common
+{
+    /// <summary>
+    /// A simple console logger - logs all incoming log messages to the console
+    /// </summary>
+    public class SimpleConsoleLogger : ILogger
```

Comment:
Should this be in a sample project rather than part of library?

Reply:
Either is possible. I added it to the library so it's easier to reuse, but one could, of course, also reuse it by copy-pasting.

Reply:
@krwq, agreed - implementations of ILogger shouldn't ship as part of the framework - we can use them in example projects to show concepts, but it shouldn't be part of the system.devices.* namespace

Reply:
This isn't going to the System.Devices namespace, just the Iot.Devices namespace. I'd rather not replicate the implementation in all examples

Reply:
Agree that I would like to see it in Iot.Device namespace. Yes it's an implementation, but bindings are as well implementations of hardware drivers :-) So why not having it there like the rest of the bindings, easy and fast to use. But clearly not in Sytem.Device